### PR TITLE
VС-4234: empty module assets, VC-4334: genesis.json

### PR DESCRIPTION
**Pull request by vovan1983**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: modules/vipcoin/chain/assets/source/local/source.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
```
Comment (alex-bodnar) [inline modules/vipcoin/chain/assets/source/local/source.go]: Удали Copyright, он не нужен

**Comment by alex-bodnar** [comment]
В файл config.tmpl.yaml в поле modules добавь vipcoin_assets

File: modules/vipcoin/chain/assets/source/local/source.go (hunk 1 above)
Comment (vovan1983) [reply modules/vipcoin/chain/assets/source/local/source.go]: Весь комментарий с Copyright в этом файле? Просто во всех файлах модуля такой Copyright есть.

Comment (alex-bodnar) [reply modules/vipcoin/chain/assets/source/local/source.go]: Во всех файлах

**Comment by IamStubborN** [comment]
LGTM

**Review by alex-bodnar — APPROVED** [review]
LGTM

**Review by IamStubborN — APPROVED** [review]

**Pull request merged**